src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -15,6 +15,10 @@ const RESERVED_CATCH_ALL_IPS = {
   ipv6: '0000::',
 };
 
+const RESERVED_DNS_LOCALHOST_IP = '127.0.0.1';
+
+const RESERVED_DNS_BROADCAST_IP = '255.255.255.255';
```

Comment:
It would be great if we could define arrays to group the different reserved addresses as that would simplify the comparisons.

Reply:
+1